pyvista/plotting/prop3D.py

```diff
@@ -0,0 +1,223 @@
+"""Prop3D module."""
+import pyvista as pv
+
+
+class Prop3D(pv._vtk.vtkProp3D):
```

Comment:
Let's properly import `_vtk` as `from pyvista import _vtk` and use that namespace directly.